setup.py

```diff
@@ -7,7 +7,7 @@
 requirements = [
     "funcy==1.16",
     "pefile==2021.9.3",
-    "vivisect==1.0.6",
+    "vivisect>=1.0.6",
```

Comment:
viv point releases often introduce breaking changes so i'd prefer to keep this pinned precisely.

Reply:
Oh, I didn't know, I tested it for both vivisect 1.0.6 and 1.0.7 and no errors were thrown.
If you agree I would like to change it to between 1.0.6 and 1.0.7

Reply:
we have some pending issues with viv-utils v1.0.6 described here: https://github.com/mandiant/capa/pull/860

then again, viv-utils master is already on v1.0.6 so lets keep moving forward. lets make the change to v1.0.7.